schemas/2015-08-01/Microsoft.Compute.json

```diff
@@ -1,549 +1,824 @@
-                    "items": {
-                        "$ref": "#/definitions/vaultCertificateUrl"
-                    }
-                }
+        "adminUsername": {
+          "type": "string"
+        },
+        "adminPassword": {
+          "type": "string"
+        },
+        "customData": {
+          "type": "string"
+        },
+        "windowsConfiguration": {
+          "$ref": "#/definitions/windowsConfiguration"
+        },
+        "linuxConfiguration": {
+          "$ref": "#/definitions/linuxConfiguration"
+        },
+        "secrets": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/secret"
+          }
+        }
+      },
+      "type": "object",
+      "required": [
+        "computerNamePrefix",
+        "adminUsername",
+        "adminPassword"
+      ]
+    },
+    "virtualMachineScaleSetOSDisk": {
+      "properties": {
+        "osType": {
+          "type": "string"
+        },
+        "name": {
+          "type": "string"
+        },
+        "vhdContainers": {
+          "type": "array",
+          "items": {
+            "type": "string"
+          }
+        },
+        "caching": {
+          "type": "string"
+        },
+        "createOption": {
+          "type": "string"
+        }
+      },
+      "type": "object",
+      "required": [
+        "name",
+        "createOption"
+      ]
+    },
+    "virtualMachineScaleSetStorageProfile": {
+      "properties": {
+        "imageReference": {
+          "$ref": "#/definitions/imageReference"
+        },
+        "osDisk": {
+          "$ref": "#/definitions/virtualMachineScaleSetOSDisk"
+        }
+      },
+      "type": "object",
+      "required": [
+        "osDisk"
+      ]
+    },
+    "virtualMachineScaleSetExtension": {
+      "properties": {
+        "name": {
+          "type": "string"
+        },
+        "properties": {
+          "properties": {
+            "publisher": {
+              "type": "string"
             },
-            "type": "object",
-            "required": [
-                "sourceVault",
-                "vaultCertificates"
-            ]
+            "type": {
+              "type": "string"
+            },
+            "typeHandlerVersion": {
+              "type": "string"
+            },
+            "settings": {
+              "type": "object"
+            }
+          },
+          "type": "object",
+          "required": [
+            "publisher",
+            "type",
+            "typeHandlerVersion"
+          ]
+        }
+      }
+    },
+    "virtualMachineScaleSetExtensionProfile": {
+      "properties": {
+        "extensions": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/virtualMachineScaleSetExtension"
+          }
+        }
+      },
+      "type": "object"
+    },
+    "ipConfiguration": {
+      "properties": {
+        "name": {
+          "type": "string"
+        },
+        "properties": {
```

Comment:
line 736 to line line 751 should under another "properties", number of "properties" should always be odd number.